docs/docs/guides/web3_migration_guide/providers_migration_guide.md

```diff
@@ -105,11 +105,12 @@ interface ReconnectOptions {
 }
 ```
 
-In 4.x, the options object is of type `ClientRequestArgs` or of `ClientOptions`. See [here](https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules__types_node_http_d_._http_.clientrequestargs.html) for `ClientRequestArgs` and [here](https://github.com/websockets/ws) for `ClientOptions`.
+In 4.x, the `socketOptions` parameter is of type `ClientRequestArgs` or of `ClientOptions`. See [here](https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules__types_node_http_d_._http_.clientrequestargs.html) for `ClientRequestArgs` and [here](https://github.com/websockets/ws) for `ClientOptions`.
 
-In 4.x a second option parameter can be given regarding auto-reconnecting, delay and max tries attempts. And here is its type:
+In 4.x the `reconnectOptions` parameter can be given regarding auto-reconnecting, delay and max tries attempts. And here is its type:
 
 ```ts
+// this is the same options interface used for both WebSocketProvider and IpcProvider
```

Comment:
could you also add one example for each provider by passing these options. That will be helpful to understand more clearly .thanks

Reply:
Added. Kindly check now.